README.md

```diff
@@ -33,6 +34,22 @@ Warning: This compatibility matrix was initially created based on experience, we
 
 Warning: By default the expressions will generate *grafana 7.2+* compatible rules using the *$__rate_interval* variable for rate functions. If you need backward compatible rules please set *grafana72: false* in your *_config*
 
+### Release steps
+
+Maintainers can run release when it's time to do so. 
+
+1. Checkout `master` branch and pull for latest.
+    ```bash
+    git checkout master
+    ```
+2. Create a tag following sem-ver versioning for the version and trigger release. 
+    ```bash
+    tag=version=${x.x.x}; git tag $tag && git push origin $tag
```

Comment:
Change to indicate required pattern and make it clear that there are 3 independent components of the version number:

```suggestion
	# replace MAJOR.MINOR.PATCH with e.g. 1.2.3
    tag=version=version-MAJOR.MINOR.PATCH; git tag $tag && git push origin $tag
```